### ISSUE TITLE
Question regarding crew & beds

### ISSUE THREAD

**Issue by Dwarf-LordPangolin** [tsunamayo/Starship-EVO-Backers-Ships#7]
While we're working on these, particularly the larger ones that will need crew eventually, should we be leaving space for beds? I've got a spot on my shuttle that'd be perfect for a couple player or NPC bunks, but I didn't know if those would be implemented soon, or what their dimensions would be when they're implemented (though I'm guessing they'd be 1x1x2).

**Comment by tsunamayo** [comment]
Yes bed will be 1x1x2, what you did was perfect, we will replace it with the official block when it is implemented! For larger ship yes it is better to leave space for that. But honestly for larger ship I think there will be ton of empty space inside, it should not be a problem to refit.
Cheers
